plugin-template

```diff
@@ -307,5 +319,23 @@ def check_for_deprecated_files(plugin_root_dir):
         print("No deprecated files found.")
 
 
+def migrate_github_issues(config, plugin_root_dir, verbose=False):
+    """Moves the plugin's issue checker CI/CD off of Redmine and onto GitHub"""
+    config['issue_tracker'] = 'github'
+    name = config['plugin_name']
+    redmine = config['redmine_project'] or name
+    config['redmine_project'] = None
+    config['update_redmine'] = False
+    write_template_section(config, "generate_config", plugin_root_dir, verbose=verbose)
+    write_template_section(config, "github", plugin_root_dir, verbose=verbose)
+    print("\nUpdating pyproject.toml to point to GitHub issues")
+    search_string = "pulp.plan.io"
+    replace_string = f"github.com\/pulp\/{config['plugin_name']}"
+    os.system(f"sed -i 's/{search_string}/{replace_string}/' {plugin_root_dir}/pyproject.toml")
+    print(f"\nPlease check the Pulp-CI PR processor (https://git.io/Jtsz5) and remove {name}"
+          f" if present")
+    print(f"\nDon't forget to migrate current issues from https://pulp.plan.io/projects/{redmine}"
+          f" and remove the repo from the project settings")
```

Comment:
do we have some script for it?
If so, can we link it here?

Reply:
This is the script I found https://github.com/IQSS/redmine2github that I was talking about in our previous meeting. I haven't tested it, but would you still like me to add a link to it?

Reply:
sorry, I forgot to reply, if it is not tested, it is better not to adding it